### PR TITLE
LPS-190168 EVP Automated Test | Master

### PR DESCRIPTION
**Pull request by joelrocha68**
[LPS-190168](https://liferay.atlassian.net/browse/LPS-190168)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 1]
```diff
@@ -63,6 +63,86 @@ definition {
 		}
 	}
 
+	@description = "LPS-180358 - Verify that the status follow the correct workflow after a finance user request more info"
+	@priority = 5
+	test CanFinanceUserVerifyStatusAfterRequestMoreInfo {
+		task ("Given that a new organization was created with employee user") {
+			User.logoutAndLoginPG(
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:70]: I think you can change this task for: `"Given There is a created Organization by an Employee user" `

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 2]
```diff
@@ -63,6 +63,86 @@ definition {
...
+				organizationSite = "Social Media Link",
+				smallDescription = "Small Description",
+				state = "State",
+				street = "Street",
+				taxId = "Tax Id",
+				zipCode = "Zip");
+		}
+
+		task ("When the Manager approved an organization") {
+			User.logoutAndLoginPG(
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:98]: I think you can change this task for: `"And this organization is with 'Awaiting finance Approval' status "`

**Review by MGabriellaS — COMMENTED** [review]
@joelrocha68  Could you take a look?

**Comment by joelrocha68** [comment]
ci:test:sf

**Pull request closed without merging**